specification/containerservice/resource-manager/Microsoft.ContainerService/aks/preview/2025-01-02-preview/managedClusters.json

```diff
@@ -8419,6 +8423,37 @@
       },
       "description": "Microsoft Defender settings for the security profile threat detection."
     },
+    "ManagedClusterSecurityProfileDefenderSecurityGating": {
+      "type": "object",
+      "properties": {
+        "enabled": {
+          "type": "boolean",
+          "description": "Whether to enable Defender security gating. When enabled, the gating feature will scan container images and audit or block the deployment of images that do not meet security standards according to the configured security rules."
+        },
+        "identities": {
+          "type": "array",
+          "description": "List of identities that the admission controller will make use of in order to pull security artifacts from the registry. These are the same identities used by the cluster to pull container images. Each identity provided should have federated identity credential attached to it.",
+          "items": {
+            "type": "object",
```

Comment:
This shouldn't be defined inline, it needs a `$ref`

Reply:
Please fix this in 2025-02-02-preview (I don't think it can be fixed in 2025-01-02-preview at this point)